doc/source/reference/c-api/ufunc.rst

```diff
@@ -10,8 +10,7 @@ UFunc API
 Constants
 ---------
```

Comment:
https://16385-908607-gh.circle-artifacts.com/0/doc/build/html/reference/c-api/ufunc.html#constants

Reply:
Looks basically identical to https://numpy.org/devdocs/reference/c-api/ufunc.html anyway